deploy/qserv.sh

```diff
@@ -0,0 +1,131 @@
+    -l, --local          Install from local git repository
+    --n NAMESPACE        Specify namespace (default: kube-system)
+    --no-kubedb          Do not install KubeDB operator
+    --uninstall          Uninstall Qserv-operator,
+                         and related CustomResourceDefinition/CustomResource
+
+  Install Qserv-operator, and eventually KubeDB-operator.
+
+EOD
+}
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+
+echo "Check kubeconfig context"
+kubectl config current-context || {
+  echo "Set a context (kubectl use-context <context>) out of the following:"
+  echo
+  kubectl config get-contexts
+  exit 1
+}
+echo ""
+
+KUBE_DB=true
+LOCAL_INSTALL=false
+UNINSTALL=false
+PURGE=true
+
+NAMESPACE=$(kubectl config view --minify --output 'jsonpath={..namespace}')
+NAMESPACE=${NAMESPACE:-default}
+export QSERV_DOCKER_REGISTRY=${QSERV_DOCKER_REGISTRY:-qserv}
+export QSERV_OPERATOR_TAG=${QSERV_OPERATOR_TAG:-v0.13.0-rc.0}
+
+
+while test $# -gt 0; do
+  case "$1" in
+    -h | --help)
+      usage
+      exit 0
+      ;;
+    -l | --local)
+      LOCAL_INSTALL=true
+      shift
+      ;;
+    -n)
+      shift
+      NAMESPACE="$1"
+      shift
+      ;;
+    --no-kubedb)
+      KUBE_DB=false
+      shift
+      ;;
+      --uninstall)
+      export UNINSTALL=true
+      shift
+      ;;
+    --purge)
+      export PURGE=true
+      shift
+      ;;
+    *)
+      echo "Error: unknown flag:" $1
+      usage
+      exit 1
+      ;;
+  esac
+done
+
+if [ "$UNINSTALL" = true ]; then
+
+  kubectl delete deployment,role,rolebinding,serviceaccount qserv-operator
+  kubectl delete crds qservs.qserv.lsst.org
+
+  (
+  # Put KubeDB backup files (yaml CRDs) inside a temporary directory
+  TMP_DIR=$(mktemp -d --suffix="-qserv-operator")
+  cd "$TMP_DIR"
+  curl -fsSL https://raw.githubusercontent.com/kubedb/installer/89fab34cf2f5d9e0bcc3c2d5b0f0599f94ff0dca/deploy/kubedb.sh | \
+  bash -s -- -n "$NAMESPACE" --uninstall --purge
+  )
+  echo
+  echo "Successfully uninstall Qserv-operator"
+  exit 0
+fi
+
+if [ "$LOCAL_INSTALL" = true ]; then
+  MANIFESTS_DIR=$(dirname "$DIR")
+else
+  MANIFESTS_DIR="https://raw.githubusercontent.com/lsst/qserv-operator/master"
+fi
+
+kapply="kubectl apply -n $NAMESPACE -f "
+
+echo "Install Qserv-operator"
+
+$kapply "$MANIFESTS_DIR"/deploy/crds/qserv_v1alpha1_qserv_crd.yaml
+$kapply "$MANIFESTS_DIR"/deploy/service_account.yaml
+$kapply "$MANIFESTS_DIR"/deploy/role.yaml
+$kapply "$MANIFESTS_DIR"/deploy/role_binding.yaml
+$kapply "$MANIFESTS_DIR"/deploy/operator.yaml
+
+if [ "$KUBE_DB" = true ]; then
+  (
+  # Put KubeDB install files inside a temporary directory
+  TMP_DIR=$(mktemp -d --suffix="-qserv-operator")
+  cd "$TMP_DIR"
+  echo "Install KubeDB"
+  # See https://kubedb.com/docs/v0.13.0-rc.0/setup/install/, but installer is broken
+  curl -fsSL https://raw.githubusercontent.com/kubedb/installer/89fab34cf2f5d9e0bcc3c2d5b0f0599f94ff0dca/deploy/kubedb.sh | \
+  KUBEDB_CATALOG=redis bash -s -- -n "$NAMESPACE"
```

Comment:
I don't really like curl'ing the kubedb script in this script. Do we want to consider checking it into the qserv-operator source, or is there another better way? 
Also, having the kubedb script fully named twice is a recipe for errors if/when changing the script. It at least wants to be assigned to a variable, or the default value of a command line arg (in case a caller wants to use a different script?)